app/views/shared/_article.html.erb

```diff
@@ -1,13 +1,13 @@
 <% category = cms_block_content(:article_type, article) %>
-<% url = article.url(:relative) %>
+<% url      = article.url(:relative) %>
```

Comment:
I added this so it's easier to read when you have multiple lines of assignment and everything aligns nicely, it's a ruby thang